integ-tests/src/main/java/com/google/sql/SqlReader.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.sql;
+
+import org.apache.commons.io.FileUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+public final class SqlReader {
+
+    private SqlReader() {}
+
+    public static String getSql(String sqlPath) {
+        try {
+            return FileUtils.readFileToString(new File(sqlPath), StandardCharsets.UTF_8);
+        } catch (IOException exception) {
+            throw new IllegalStateException("Error reading for sql file [" + sqlPath + "]", exception);
```

Comment:
I'd prefer `String.format` over string concatenation as it makes the formatting string easier to read and less error prone.

Reply:
Agree. Fixed.